modules/ivc_champva/app/jobs/ivc_champva/missing_form_status_job.rb

```diff
@@ -18,16 +18,44 @@ def perform
       # Send the count of forms to DataDog
       StatsD.gauge('ivc_champva.forms_missing_status.count', forms.count)
 
-      # Send each form UUID to DataDog
+      current_time = Time.now.utc
       forms.each do |form|
+        # Check if we've been missing Pega status for > custom threshold of days:
+        elapsed_days = (current_time.to_i - form.created_at.to_i) / (60 * 60 * 24)
+        threshold = Settings.vanotify.services.ivc_champva.failure_email_threshold_days.to_i || 7
+        if elapsed_days >= threshold && !form.email_sent
+          template_id = "#{form[:form_number]}-FAILURE"
+          send_failure_email(form, template_id)
+          StatsD.increment('ivc_champva.form_missing_status_email_sent', tags: ["id:#{form.id}"])
+        end
```

Comment:
In the future you will want to change this to use your new Monitor Class or something of that ilk, just as a heads up

Reply:
Noted - going to plan to tackle that in a followup PR. Thanks!